matrix_client/api.py

```diff
@@ -238,6 +238,16 @@ def send_message(self, room_id, text_content, msgtype="m.text"):
             self.get_text_body(text_content, msgtype)
         )
 
+    def send_html(self, room_id, html_content, msgtype="m.text"):
+        """Send an html formatted message.
+
+        Args:
+            room_id (str): The room ID to send the event in.
+            html_content (str): The html formatted message to be sent.
+        """
+        return self.send_message_event(room_id, "m.room.message",
+                                       self.get_html_body(html_content, msgtype))
```

Comment:
So the use case is if I call send_html() and my html doesn't convert to plain text very well or if I want to use my own function for doing so. Essentially, I don't believe it's a good idea to auto generate it. 

Examples would be a table usually doesn't translate very well from html to plaintext, or lists.

>This is pretty confusing since body and html_content should be mutually exclusive, but what happens if send_html is called with both parameters?

I didn't read the question very well. Sorry, you need to have a body with your html_content to describe it to clients that don't support html. It's a required field.